recipes/thermotools/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "thermotools" %}
+{% set version = "0.2.3" %}
+{% set sha256 = "8cd05a705665d9e2da2c1d05b19278945af9927dda7b4edc2c1bf0b1b96d4d71" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - cython
+    - numpy x.x
+  run:
+    - python
+    - numpy x.x
+    - scipy
+    - msmtools >=1.1.4
+
+test:
+  imports:
+    - thermotools 
+
+about:
+  home: http://github.com/markovmodel/thermotools
+  license: LGPL 
+  license_file: COPYING.LESSER
+  summary: 'Lowlevel implementation of (transition-based and histogram) reweighting analyis methods for Python.'
+
+  description: |
+    The thermotools package is a lowlevel implementation toolbox for the analyis
+    of multi-ensemble calculations. It contains estimators for the state-continuous transition-based
+    reweighting analysis method (TRAM) and its state-discrete variant (dTRAM), Bennet acceptance ratio
+    (BAR) and its multi-state variant (MBAR), and the weighted histogram analysis method (WHAM).
+    While you can use thermotools on its own, we recommend to use it in combination with PyEMMA.
+  dev_url: https://github.com/markovmodel/thermotools
+
+extra:
+  recipe-maintainers:
+    - marscher
+    - fabian-paul 
+    - cwehmeyer 
```

Comment:
These are my colleagues, so I think it is okay, if they have permissions on the feedstock ;)